.concourse/validate-pr.yaml

```diff
@@ -0,0 +1,77 @@
+---
+
+# s3, git, time, github-release, registry-image,
+# pool - serialize access to external systems
+# git - can also push
+
+resource_types:
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+resources:
+#  - name: istio-installer #DO NOT SET TO FORK - for testing use another pipeline and pause this one
+#    type: git
+#    source:
+#      uri: "https://github.com/istio/istio-installer"
+#
+  - name: istio-master
+    type: git
+    source:
+      uri: "https://github.com/istio/istio"
+      branch: master
+
+  - name: istio-installer-pr
+    type: pull-request
+    source:
+        repository: "istio/istio-installer"
+        access_token: ((GITHUB_ACCESS_TOKEN))
+
+jobs:
+  - name: validate-istio-installer-pr
+    serial_groups: [ cluster ]
```

Comment:
I think `serial_groups` only work within a pipeline [see here](https://github.com/concourse/concourse/issues/2691). So we should go with one pipeline or use the `pool` resource.